test/pdfs/Contestation-Transaction-Carte-Bancaire.pdf.link

```diff
@@ -0,0 +1 @@
+https://web.archive.org/save/https://www.hsbc.fr/content/dam/hsbc/fr/docs/pib/Contestation-Transaction-Carte-Bancaire.pdf
```

Comment:
This doesn't look like a working Internet Archive URL, and the tests failed as a result.

You'd need to use e.g. https://web.archive.org/web/20210607145115/https://www.hsbc.fr/content/dam/hsbc/fr/docs/pib/Contestation-Transaction-Carte-Bancaire.pdf instead (also with a new-line at the end to stop GitHub complaining), but given that it's already cached on the bots things might not work with only that change.
Hence the simplest solution might be to re-name this file to `test/pdfs/hsbc.pdf.link` instead, and update the manifest as well.